components/table/Table.tsx

```diff
@@ -409,7 +404,9 @@ function Table<RecordType extends object = any>(props: TableProps<RecordType>) {
   }
 
   // Indent size
-  mergedExpandable.indentSize = mergedExpandable.indentSize || indentSize || 15;
+  if (mergedExpandable.indentSize === null || mergedExpandable.indentSize === undefined) {
```

Comment:
maybe use `typeof xxx === 'number'` is a more evident solution?

Reply:
Updated thanks to your comment